examples/fluent-bit/multi-config-support/README.md

```diff
@@ -0,0 +1,80 @@
+## FireLens Example: Multiple Config support - using the Fluent Bit image with init tag
+
+This example shows you how to set multiple config files for Fluent Bit on ECS. 
```

Comment:
Let's link the use case guide here as well and say it explains how and why init was created

Reply:
Hmm maybe I need to merge use case guide first, get the real link and then add the link in it..
Actually linking them to each other..

Reply:
You can predict what the link will be based on the existing links and your new file name